### ISSUE TITLE
Disable fusing batch normalization layers into convolutional layers

### ISSUE THREAD

**Issue by Mypathissional** [onnx/onnx-tensorflow#970]
Hi,

I am trying to convert efficieinetnet-v2 from Tensorflow to Onxx and then to another framework. And I have noticed that most of the batch normalization layers are fused within the convolutional layers as a result of the constant folding. I am wondering whether it is possible to export the batch norm layers as well since I want to have a trainable model in another framework.

Any help would be appreciated,
Best Regards,
Maria

**Comment by chinhuang007** [comment]
@Mypathissional This looks like a question for the tf2onnx converter, https://github.com/onnx/tensorflow-onnx. You might want to open an issue there. Thanks.
